docs/readthedocs/source/doc/Chronos/Howto/how_to_use_benchmark_tool.md

```diff
@@ -0,0 +1,135 @@
+
+### Model
+The tool provides several built-in time series forecasting models, including TCN, LSTM, Seq2Seq, NBeats and Autoformer. To specify which model to use, run benchmark tool with `-m/--model`. If not specified, TCN is used as the default.
+```bash
+benchmark-chronos -m lstm -l 96 -o 720
+```
+
+### Stage
+Regarding a model, training and inference stages are most concerned. By setting `-s/--stage` parameter, users can obtain knowledge of throughput during training (`-s train`), throughput during inference (`-s throughput`) and latency of inference (`-s latency`). If not specified, train is used as the default.
+```bash
+benchmark-chronos -s latency -l 96 -o 720
+``` 
+
+### Dataset
+Several built-in datasets can be chosen, including nyc_taxi and tsinghua_electricity. If users are with poor Internet connection and hard to download dataset, run benchmark tool with `-d synthetic_dataset` to use synthetic dataset. Default to be tsinghua_electricity if `-d/--dataset` parameter is not specified.
+```bash
+benchmark-chronos -d nyc_taxi -l 96 -o 720
+```  
+
+### Framework
+Pytorch and tensorflow are both supported and can be specified by setting `-f torch` or `-f tensorflow`. But Autoformer does not support tensorflow backend now. The default framework is pytorch.
+```bash
+benchmark-chronos -f tensorflow -l 96 -o 720
+```
+
+### Core number
+By default, the benchmark tool will run on all physical cores. And users can explicitly specify the number of cores through `-c/--cores` parameter.
+```bash
+benchmark-chronos -c 4 -l 96 -o 720
+```
+
+### Lookback
+Forecasting aims at predicting the future by using the knowledge from the history. The required option `-l/--lookback`corresponds to the length of historical data along time.
+```bash
+benchmark-chronos -l 96 -o 720
+```
+
+### Horizon
+Forecasting aims at predicting the future by using the knowledge from the history. The required option `-o/--horizon`corresponds to the length of predicted data along time.
+```bash
+benchmark-chronos -l 96 -o 720
+```
+
+## Advanced Options
+Besides, number of processes and epoches can be set by `--training_processes` and `--training_epochs`. Users can also tune batchsize during training and inference through `--training_batchsize` and `--inference_batchsize` respectively.
+```bash
+benchmark-chronos --training_processes 2 --training_epochs 3 --training_batchsize 32 --inference_batchsize 128 -l 96 -o 720
+```
+
+In order to benchmark the performance with accelerator, run the benchmark tool with `--ipex` enabled, then use intel-extension-for-pytorch as accelerator for trainer. If want to use quantized model to predict, just run the benchmark tool with `--quantize` enabled and the quantize framework can be specified by setting `--quantize_type`. Pytorch_fx, pytorch_ipex, onnxrt_qlinearops and openvino are supported currently while default quantize framework is pytorch_fx.
```

Comment:
I think that interchanged between L75 and L70 will be better, explain the `inference_framework` firstly, and then `--quantize` and `--quantize_type`. After that, in the end of L70, maybe we can explain as following:

The `--quantize_type` need to be set, only when you want to use pytorch_ipex as quantize type. Otherwise, defaut quantize type will be selected according to `inference_framework`. If the `torch` is the inference framework, then the pytorch_fx will be the default quantize type. If you choose the `onnx` as your inference framework, the onnxrt_qlinearops will be your quantize type. If the `openvino` is your inference framework, the openvino quantize type will be selected.

Reply:
good idea

Reply:
done